tests/phpunit/api/v3/GeometryTest.php

```diff
@@ -271,17 +292,20 @@ public function testRemoveOnlyCollection() {
    * Remove a Geometry from an collection when the Geometry is in Multiple collections.
    */
   public function testRemoveCollection() {
+    // Create Upper house Geometry
     $collectionParams = [
       'label' => 'Tasmanian Upper House',
       'source' => 'TasEC',
       'geometry_collection_type_id' => $this->externalCollectionType['id'],
     ];
     $collection = $this->callAPISuccess('GeometryCollection', 'create', $collectionParams);
+    // Create 2nd collection more broader than just about tasmanian upper house
```

Comment:
How about "Create a superset of the first collection", which is what you're trying to do.  That it's Tassie is irrelevant.